tools/tpm2_changepps.c

```diff
@@ -0,0 +1,68 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#include "log.h"
+#include "tpm2.h"
+#include "tpm2_options.h"
+#include "tpm2_tool.h"
+
+typedef struct dictionarylockout_ctx dictionarylockout_ctx;
+struct dictionarylockout_ctx {
+    struct {
+        const char *ctx_path;
+        const char *auth_str;
+        tpm2_loaded_object object;
```

Comment:
you just need a tpm2_session * for the platform auth. You don't need an object, because the object is fixed.

Reply:
True. Simply passing the session structure now.